File: RevenueCalculator.scala
object RevenueCalculator {
  def calculateRevenue(user: String, amount: Double): Double = {
    val tax = 0.2
    val total = amount + (amount * tax)
    val unused = "dead"
    total
  }
  def main(args: Array[String]): Unit = {
    val result = calculateRevenue("alice", 1000)
    println("Revenue: " + result)
  }
}

File: RevenueCalculator.java
public class RevenueCalculator {
    public double calculateRevenue(String user, double amount) {
        double tax = 0.18;
        double total = amount + (amount * tax);
        String unused = "not used";
        return total;
    }
    public static void main(String[] args) {
        RevenueCalculator rc = new RevenueCalculator();
        double result = rc.calculateRevenue("alice", 1000);
        System.out.println("Revenue: " + result);
    }
}
